cassandra-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/CassandraCluster.java

```diff
@@ -569,6 +569,11 @@ private CassandraNode buildCassandraNode(Protos.Offer offer, boolean seed, Strin
         CassandraNode.Builder builder = CassandraNode.newBuilder()
                 .setHostname(offer.getHostname())
                 .setTargetRunState(CassandraNode.TargetRunState.RUN)
+                .addDataVolumes(
+                    DataVolume.newBuilder()
+                        .setPath(configuration.getDefaultConfigRole().getPreDefinedDataDirectory())
+                        .build()
+                )
```

Comment:
I think we need to keep track of it now because there is the potential that we will need to support migrating the data from a previously defined directory to a persistent volume.